frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -183,14 +183,16 @@ function VirtualKey:onUnfocus()
     self[1].inner_bordersize = 0
 end
 
-function VirtualKey:onTapSelect()
+function VirtualKey:onTapSelect(skip_flash)
     if self.flash_keyboard and not self.skiptap then
         self[1].inner_bordersize = self.focused_bordersize
         self:update_keyboard(false, true)
         if self.callback then
             self.callback()
         end
-        UIManager:tickAfterNext(function() self:invert(false) end)
+        if not skip_flash then
```

Comment:
Seeing this in the diff made me realize it should just be part of the top condition.

The flashing behavior of this whole popup stuff isn't quite optimized in any case, with the whole keyboard refreshing and all.